zio-kafka/src/main/scala/zio/kafka/consumer/internal/LiveCommitter.scala

```diff
@@ -23,6 +23,12 @@ private[consumer] final class LiveCommitter(
   pendingCommits: Ref.Synchronized[Chunk[Commit]]
 ) extends Committer {
 
+  override val markCommittedInTransaction: Map[
+    TopicPartition,
+    OffsetAndMetadata
+  ] => Task[Unit] = offsets =>
+    committedOffsetsRef.modify(_.addCommits(Chunk(Commit(java.lang.System.nanoTime(), offsets, null)))).unit
```

Comment:
Let's put this below `val commit` to keep the order of the trait.

```suggestion
    committedOffsetsRef
      .modify {
        // The continuation promise can be `null` because this commit is not handled by the consumer,
        // and transactional commits have no completion handler anyway.
        _.addCommits(Chunk(Commit(java.lang.System.nanoTime(), offsets, null)))
      }
      .unit
```

Reply:
I updated the proposed comment.